vulture/core.py

```diff
@@ -201,14 +237,27 @@ def get_set(typ):
 
         self.ignore_names = ignore_names or []
         self.ignore_decorators = ignore_decorators or []
+        self.noqa_matches = defaultdict(list)
 
         self.filename = ""
         self.code = []
         self.found_dead_code_or_error = False
 
+    def add_noqa_match(self, code, lineno):
+        self.noqa_matches[code].append(lineno)
+
+    def parse_noqa(self, code):
+        for index, line in enumerate(code):
+            match = _find_noqa(line)
+            if match is not None:
+                code = match.groupdict()["codes"] or "all"
```

Comment:
This needs to handle the case when multiple codes are given. (split by commas and strip whitespace, I guess.)

Reply:
Done!